apps/web/assets/js/components/Home.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Header from './Header';
+import Footer from './Footer';
+import Filters from './Filters';
+import List from './List';
+
+export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      filters: {
+        levels: [1, 5, 9],
+      },
+    };
+
+    this.handleFilterChange = this.handleFilterChange.bind(this);
+  }
+
+  handleFilterChange(filters) {
+    this.setState(prevState => ({
+      filters: filters,
+    }));
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
```

Comment:
This looks like it's probably an extra `div`

Reply:
Will do didn't want to mess with the brunch config as I've never used it before didnt seem like an ideal solution to revert syntax ill revert those changes and get rid of the extra div